addon/initializers/active-model-adapter.ts

```diff
@@ -0,0 +1,12 @@
+import Application from '@ember/application';
+import ActiveModelAdapter from 'active-model-adapter';
+import ActiveModelSerializer from 'active-model-adapter/active-model-serializer';
+
+export function initialize(application: Application): void {
+  application.register('adapter:-active-model', ActiveModelAdapter);
+  application.register('serializer:-active-model', ActiveModelSerializer);
```

Comment:
we should probably just kill this initializer. Alternative for auto registration is to re-export in the app directory this serializer and adapter at the path `adapters/-active-model` and `adapters/-active-serializer` though the preceding dash is a troll and we probably should remove anywhere that it is depended on.

Reply:
Mind if we do this in another PR? This PR is already kind of enormous.